### DISCUSSION TITLE
Inconsistent marker results from the previous version

### DISCUSSION THREAD

**Discussion by nesetozel** [stuart-lab/signac#728] [Q&A] [answered]
Hi @timoast 

Sorry to bother you twice in two days. I have been using Signac 1.1.0 until very recently when I updated to 1.3.0. I had previously calculated marker peaks for an object (happy to share if it helps), like so: 

markers <- FindAllMarkers(P48.Tm, assay = 'peaks',only.pos = FALSE,
                          test.use = 'LR',latent.vars = 'nCount_peaks')
DApeaks <- unique(markers$gene)

This had resulted in 3240 unique markers. However, running the same code with the same object in the new version only gives ~1100 unique markers, which is a subset of the previous one. Version 1.2.1 gives the same result as 1.1.0 so I'm assuming this has something to do with this change in the latest version:
_Added FoldChange() method for ChromatinAssay() object that sets proper parameters for chromatin data. This fixes the calculation of fold changes when running Seurat::FindMarkers() on single-cell chromatin data._

If this was a bug fix, and the extra ~2k peaks I was getting before should not have been markers, does that mean everything I did with the previous marker peak set is now invalid? Thanks very much!

Best,
Neset

**Comment by timoast** [comment, marked as answer]
It means that the fold changes from version <1.3.0 were not accurate, as it was computed using the tf-idf values. The p values should not change.

**Reply by nesetozel** [reply]
So it's safe to assume that the markers I had before were in fact still differentially accessible (since they passed the p-value threshold), even though their real fold change might have been below Seurat's default minimum logFC (0.25)?

**Reply by timoast** [reply]
You typically want to also filter on fold-change, not only the p-value. I'd suggest taking your "significant" peaks and sorting by fold-change, and taking a look at the ones with the lowest fold change using the `CoverageBrowser()` function

**Reply by nesetozel** [reply]
Thanks!
